corehq/apps/export/models/new.py

```diff
@@ -1240,8 +1240,13 @@ def _process_app_build(cls, current_schema, app, form_xmlns):
 
         case_updates = form.get_case_updates(form.get_module().case_type)
         xform = form.wrapped_xform()
+        if isinstance(form.actions, AdvancedFormActions):
+            open_case_actions = form.actions.open_cases
+        else:
+            open_case_actions = form.actions.subcases
+
         repeats_with_subcases = {
-            subcase_action for subcase_action in form.actions.subcases
+            subcase_action for subcase_action in open_case_actions
```

Comment:
noting that `AdvancedForm` has a `get_open_subcase_actions` function, so depending on the intent you might want to use that instead (otherwise renaming the variable would be more clear)